wings/wallet/ethereum/web3_wallet.pyx

```diff
@@ -14,12 +14,12 @@ from hummingbot.core.clock cimport Clock
 from wings.events import (
     WalletEvent
 )
-from wings.ethereum_chain import EthereumChain
+from wings.wallet.ethereum.ethereum_chain import EthereumChain
```

Comment:
It sounds like to me that this file should be deleted, not merely changed.

Reply:
fixed